Core/HLE/sceUmd.cpp

```diff
@@ -495,7 +504,9 @@ void __UmdReplace(std::string filepath) {
 		}
 	}
 	delete currentUMD;
-
+	UMD_insterted = false;
+	sleep_ms(200); // Wait sceUmdCheckMedium call
```

Comment:
A better solution may be to schedule an event (you know, RegisterEvent and so on) that will set UMDInserted back after a little bit.